libs/ledgerjs/packages/hw-transport-node-speculos-http/src/SpeculosHttpTransport.ts

```diff
@@ -51,8 +67,10 @@ export default class SpeculosHttpTransport extends Transport {
         .then((response) => {
           response.data.on("data", (chunk) => {
             log("speculos-event", chunk.toString());
-            // XXX: we could process display events here
-            // client side automation or UI tests/checks
+            const split = chunk.toString().replace("data: ", "");
+            // FIXME: ugly patch because speculos return us a `data: `
```

Comment:
Comment below code again? :smile:

Reply:
I don't know why I like comment below so much ...

Reply:
to me it's not an ugly thing, i think it's the spec of Event Stream that do this 🤔 
https://html.spec.whatwg.org/multipage/server-sent-events.html

so i think you can remove the comment.